### ISSUE TITLE
feat: ✨ footer template

### ISSUE THREAD

**Issue by fabian-paus** [synergy-design-system/synergy-design-system#501]
# Checklist (for the issue creator)

- [x] I have looked at the documentation site and was not able to find the component.
- [x] I have looked into the Figma boards and was not able to find the component.
- [ ] I would be willing to provide a PR for this component.

# Story

As an application or website developer, I want to add a footer that includes all the required legal information. Since every website and application is legally required to include this, the footer should look the same (or similar) on all SICK websites. Otherwise, everyone will make their own footer that behaves and looks differently.

Example footers:
- https://www.sick.com/de/en/
- https://id.sick.com
- https://cloud.sick.com/

What the components needs to do:
- Be displayed at the bottom of the page
- Include links to: 
  - [Imprint](https://www.sick.com/de/en/w/imprint)
  - [Terms and conditions](https://www.sick.com/de/en/w/tac)
  - [Terms of Use](https://www.sick.com/de/en/w/terms-of-use)
  - [Privacy Policy](https://www.sick.com/de/en/w/dataprotection)
- Include a copyright notice
- Provide translations for our 18 or so supported languages (or be translatable)
- Reactive layout that adapts to different resolutions (e.g. sick.com changes the position of the copyright notice depending on the size of the viewport)

![image](https://github.com/synergy-design-system/synergy-design-system/assets/11577359/ede298d5-e03c-4630-a7d5-dcab8ed3dd64)

Alternative design: Provide a way to add links to the footer from the application level. Then, synergy does not need to provide the translations.

Application context: Every website or application that is customer facing needs this at the bottom of the page for legal reasons.

# Acceptance criteria

- [ ] All DoD items successfully completed
- [ ] A footer template is included into the synergy library (AppShell)
- [ ] Add to FW demos
- [ ] Dog footing of link styling
- [ ] Optionally add footer style into styles-package (background color, paddings,...)
- [ ] add responsiveness

**Comment by fabian-paus** [comment]
We can provide the translations for the footer link texts (SICK ID team) if that is necessary. Depending on whether you want to include them in synergy.

**Comment by klaarseSICKAG** [comment]
Hi Fabian,
thanks for you input.

We already discussed the footer topic in the team and decided to go for a template approach instead of having a component. Templates are more flexible and provide a copy & paste approach which can be adapted better to application needs.
Regarding translation: we decided against translating text inside synergy. Otherwise we have to provide translations for each language and we cannot know which languages are necessary for an application. Also translations may come from other sources (like the CMS on the website).
Nevertheless your Input is valuable and we certainly take your requirements into account.
